### PR TITLE
DO NOT MERGE

### PR DESCRIPTION
**Pull request by fizal619**
Use this as a point of reference I guess? 
**View in split mode for best effect.

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
```
Comment (fizal619) [inline src/components/App/App.jsx:1]: I just took out things that were causing warning to help with my debugging.

File: src/components/App/App.jsx [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 import './App.css';
 
 import Menu from '../Menu/Menu'
-import Items from '../Items/Items'
```
Comment (fizal619) [inline src/components/App/App.jsx:5]: One of the things I removed from App.jsx to help suppress.

File: src/components/App/App.jsx [hunk 3]
```diff
@@ -11,8 +10,7 @@ class App extends Component {
   constructor(props) {
     super(props);
     this.state={
-      name: props.items.name,
-      price: props.items.price
+      items: props.items   //here it stays the exact same array we passed in.
```
Comment (fizal619) [inline src/components/App/App.jsx:13]: renaming these to just the object being passed in

Comment (fizal619) [reply src/components/App/App.jsx:13]: so right now it's still
the original object stored in `this.state.items`

File: src/components/App/App.jsx [hunk 4]
```diff
@@ -22,9 +20,8 @@ class App extends Component {
       <div className="App">
         <div className="menu-header">
           <h2>Welcome to Jordan Sassy Menu </h2>
-          <Menu />
-          <Items name={this.props.items.name} />
-          <Items price= {this.props.items.price} />
+          <Menu items={this.state.items} />
```
Comment (fizal619) [inline src/components/App/App.jsx:23]: passed it into menu

File: src/components/App/App.jsx [hunk 5]
```diff
@@ -22,9 +20,8 @@ class App extends Component {
       <div className="App">
         <div className="menu-header">
           <h2>Welcome to Jordan Sassy Menu </h2>
-          <Menu />
-          <Items name={this.props.items.name} />
```
Comment (fizal619) [inline src/components/App/App.jsx:26]: You were probably troubleshooting with these. I took em out cause it isn't needed until later.

File: src/components/Items/Items.jsx [hunk 6]
```diff
@@ -3,7 +3,6 @@ import React from 'react';
 
 export default (props) => {
   return <div className="Menu_Items">
-    <Items name={props.items.name} />,
-    <Items price={props.items.price} />
+    <p> {props.name}, {props.price} </p>
```
Comment (fizal619) [inline src/components/Items/Items.jsx:6]: Here, you were trying to use "Items" but this is in `Items.jsx`. Essentially you were on the right track, but you can simply just return a single div or something containing the information we need.

File: src/components/Menu/Menu.jsx [hunk 7]
```diff
@@ -1,11 +1,9 @@
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
 
-import Message from '../Items/Items';
+import Items from '../Items/Items';
```
Comment (fizal619) [inline src/components/Menu/Menu.jsx:3]: renamed this

File: src/components/Menu/Menu.jsx [hunk 8]
```diff
@@ -1,11 +1,9 @@
...
 
 export default class Menu extends Component {
-  constructor(props) {
-    super(props);
-  }
+
```
Comment (fizal619) [inline src/components/Menu/Menu.jsx:6]: the constructor was removed to help suppress errors, ignore this one

**Review by fizal619 — COMMENTED** [review]
@yaelber 
I hope this helps, you were definitely on the right track. You just lost what you were trying to pass along; along the way. 

Let's make sure we sit down at 5:45 to talk about it. 😄 

Cheers!

File: src/components/App/App.jsx (hunk 2 above)
Comment (yaelber) [reply src/components/App/App.jsx:5]: So Items/Items shouldn't be imported?

File: src/components/Items/Items.jsx (hunk 6 above)
Comment (yaelber) [reply src/components/Items/Items.jsx:6]: I guess what I'm not getting as when to use a react tag and when to use a regular html tag

**Pull request closed without merging**